File: src/domain/labels.ts
// Single source of truth for UI-facing labels.
// Internal keys stay generic; this map translates them to V5 game terms.

export const LABELS = {
  thirst: 'Hunger',
  morality: 'Humanity',
  marks: 'Stains',
  health: 'Health',
  willpower: 'Willpower',
  superficial: 'Superficial',
  aggravated: 'Aggravated',
  conditions: 'Conditions',
  customTrackers: 'Custom Trackers',
  shortNotes: 'Session Notes',
} as const;

export type LabelKey = keyof typeof LABELS;

File: android/app/src/main/kotlin/io/crimsonledger/domain/Labels.kt
package io.crimsonledger.domain

/**
 * Single source of truth for V5-flavoured UI labels. Keep this the only
 * translation surface between generic engine names and the player-facing
 * language (so swapping presets or adding i18n later is cheap).
 */
object Labels {
    const val HUNGER = "Hunger"
    const val HUMANITY = "Humanity"
    const val STAINS = "Stains"
    const val HEALTH = "Health"
    const val WILLPOWER = "Willpower"
    const val CONDITIONS = "Conditions"
    const val CUSTOM = "Custom trackers"
    const val NOTES = "Short notes"
    const val SUPERFICIAL = "Superficial"
    const val AGGRAVATED = "Aggravated"
    const val IMPAIRED = "Impaired"
    const val TORPOR = "Torpor"
    const val DEGENERATION_RISK = "Degeneration risk"
}
